index.html

```diff
@@ -12,7 +12,7 @@
   </head>
```

Comment:
- [ ] [OPTIONAL] The title is good but I would recommend you to use a brief title such as `Wek Developer at Jabscan ` or `Mark's portfolio`.

Reply:
noted